hera_qm/utils.py

```diff
@@ -264,6 +265,8 @@ def get_metrics_ArgumentParser(method_name):
         ap.add_argument('--a_priori_flag_yaml', default=None, type=str,
                         help=('Path to a priori flagging YAML with frequency, time, and/or '
                               'antenna flagsfor parsable by hera_qm.metrics_io.read_a_priori_*_flags()'))
+        ap.add_argument('--a_apriori_times_and_freqs', default=False, action="store_true",
```

Comment:
Looks like you need to add YAML flags to day threshold run argparser.

Reply:
Thanks for catching this. I've updated the parser.